tests/unit/plugins/janitor/test_authority.py

```diff
@@ -232,3 +232,31 @@ def test_create_msgs_bad_json(caplog, fake_authority):
     results = fake_authority._create_msgs([partial_instance])
     assert [] == results
     assert 1 == len(caplog.records)
+
+
+@pytest.mark.parametrize('whitelist,expected', [
+    (['whitelisted_1', 'whitelisted_1'], ['whitelisted_1', 'whitelisted_1']),
```

Comment:
It might be a good idea to use two unique names to illustrate what's happening. I realize this will work (and since it's parametrized we could leave this one around too to show that uniqueness isn't important), but it's harder to immediately tell what's going on from looking at this case.

Reply:
derp forgot to rename my test values...